src/views/Restoration.vue

```diff
@@ -208,6 +222,14 @@ export default class Restoration extends Vue {
       // fetch entity snapshot
       const entitySnapshot = await this.fetchEntitySnapshot()
 
+      // set Applicant info in entitySnapshot
+      const stateFiling = entitySnapshot.businessInfo.stateFiling
+      const filing = stateFiling && await LegalServices.fetchFiling(stateFiling)
+      entitySnapshot.orgPersons = filing.restoration.parties as OrgPersonIF
```

Comment:
1. You need to handle the case where filing, or restoration, or parties is undefined/null.

2. `orgPersons` should be of type array.

3. This code will assign ALL people from the state filing into the local store. I think you should verify that you have found the applicant and only add that one to the store. Thoughts?

Suggestion:
``

Reply:
I refactored as you suggested. Please let me know if looks okay.